aleph-client/src/contract/mod.rs

```diff
@@ -59,23 +59,42 @@ use crate::{
     AccountId, Balance, ConnectionApi, SignedConnectionApi, TxStatus,
 };
 
+/// Default gas limit, which allows up to 25% of block execution time.
```

Comment:
Where can I check that this is indeed 25% ?

Reply:
our block has 10^12 weight capacity - it should be somewhere in `primitives`

Reply:
Hmm, from what I found `MAX_BLOCK_WEIGHT` is 400 * 10^9. Maybe I should look for another constant?

Reply:
From what I understand we have 40% of the block for production, so new default would be >60% of production time

Reply:
So from what I understand it is indeed 25% of total block time, but >60% of actual max gas limit

Reply:
yes, that's right

Reply:
I've changed the docstring to be more precise